data/actionsmap/yunohost.yml

```diff
@@ -226,6 +226,97 @@ user:
                 username:
                     help: Username or email to get information
 
+#############################
+#           Alias            #
+#############################
+alias:
+    category_help: Manage alias
+    actions:
+
+        ### alias_list()
+        list:
+            action_help: List existing aliases
+            api: GET /alias
+            configuration:
+                authenticate: all
+                authenticator: ldap-anonymous
+
+        ### alias_create()
+        create:
+            action_help: Create a new alias
+            api: POST /alias
+            configuration:
+                authenticate: all
+            arguments:
+                alias:
+                    help: The unique alias to create
+                    metavar: ALIAS
+                    extra:
+                        required: True
+                        pattern: &pattern_alias
+                            - !!str ".*@.+"
+                            - "pattern_alias"
+                -f:
+                    full: --mailforward
+                    help: List of email to forward to, separated by commas without space
+                    metavar: MAIL
+                    extra:
+                        ask: ask_alias_mailforward
+                        required: True
+                        pattern: *pattern_email
+
+        ### alias_update()
+        update:
+            action_help: Update an existing email alias
+            api: PUT /alias/<alias>
+            configuration:
+                authenticate: all
+            arguments:
+                alias:
+                    help: Alias to update
+                    metavar: ALIAS
+                    extra:
+                        required: True
+                        pattern: *pattern_alias
+                --add-mailforward:
+                    help: List of mailforward addresses to add, separated by commas without space
```

Comment:
actually `--add-mailforward` accpets a space separated list of email, which breaks the ability to do `yunohost alias update --add-mailforward forward@domain.com existing-alias@domain.com`, because both addresses are considered as mailforward